apps/site/src/_pages/docs/5_spec/02_graph.mdx

```diff
@@ -163,9 +163,13 @@ Blocks may also add an `examples` key to `block-metadata.json` to define example
 A block using the graph module MAY expand `block-metadata.json` to specify:
 
 - `schema`: the versioned URL of the **entity type** associated with the block entity. As per the specification, the schema hosted at the remote address should not be subject to change and versions of entity types should be treated as immutable, permanently available, records.
-- `examples`: an array of objects conforming to the block's entity type which provide examples of the data that can be passed to it
 - `variants`: an array of objects, each with a `name` and `properties`, and optionally `description`, `icon`, and `examples`. These objects represents different variants of the block that the user can create, where `properties` sets the starting properties. As a simple example, a ‘header’ block might have variants with the `name` ‘Heading 1’ and ‘Heading 2’, which start with `{ "https://blockprotocol.org/@alice/types/property-type/level/": 1 }` and `{ "https://blockprotocol.org/@alice/types/property-type/level/": 2 }` as `properties`, respectively
-  If more complex example data is required, a block’s package may also include a file at the root named `example-graph.json`, which MUST be a JSON object, and MUST contain an `entities` key where each element is a JSON representation of an entity which MUST conform to the structure specified in [entity definition](#structure-specification).
+
+A block using the graph module MUST include an `example-graph.json` file alongside its metadata. The file's contents
```

Comment:
What's the thinking for moving the provision of example data from `MAY` to `MUST`? I'm not totally against it, but it is imposing another requirement on block authors and we should explicitly consider any new requirements.

Reply:
Oops this is currently inconsistent. [Previous thread for discussion](https://github.com/blockprotocol/blockprotocol/pull/1223#discussion_r1149557972)

Reply:
[Make the example-graph file optional](https://github.com/blockprotocol/blockprotocol/pull/1223/commits/858ae83f0f31420fad60ad094d8b0cf49b9d006a)